openff/toolkit/utils/openeye_wrapper.py

```diff
@@ -811,8 +811,10 @@ def _assign_aromaticity_and_stereo_from_3d(self, offmol):
         return offmol_w_stereo_and_aro
 
     def enumerate_protomers(
-        self, molecule: "Molecule", max_states: int = 10
-    ) -> List["Molecule"]:
+        self,
+        molecule: "Molecule",
+        max_states: int = 10,
```

Comment:
I think with this change max_states should be 0 by default, or at least a number much bigger than 10.

```suggestion
        max_states: int = 0,
```

I bet if we leave it at 10, this issue comes up again and again from people who don't know about max_states (like me until 10 minutes ago). OE seems to be able to put thousands of protomers together in only a few seconds:

```python
>>> %%timeit
... 
... molecule = Molecule.from_smiles("C(C(=O)O)(C(=O)O)C(C(=O)O)(C(=O)O)C(C(=O)O)(C(=O)O)C(C(=O)O)(C(=O)O)C(C(=O)O)(C(=O)O)")
... 
... 
... options = oequacpac.OEFormalChargeOptions()
... options.SetMaxCount(0)
... 
... molecules = []
... 
... oemol = molecule.to_openeye()
... for protomer in oequacpac.OEEnumerateFormalCharges(oemol, options):
...     molecules.append(
...         Molecule.from_openeye(
...             protomer,
...             allow_undefined_stereo=True,
...         )
...     )
2.45 s ± 47.2 ms per loop (mean ± std. dev. of 7 runs, 1 loop each)
>>> len(molecules)
6400
```

Reply:
I agree with this - The max states should be set to 0 (unlimited) by default. I'm a bit surprised that limiting this is an option at all (if we hit the max, which ones get returned? Users are guaranteed to be surprised) but I'd rather keep this PR at "behavior change" and not "API break", so just changing the default here should be good.  Please also record this behavior change in the changelog.